### DISCUSSION TITLE
Calrification-Mocha Vs Jasmine

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/codemod#100] [General]
Hi, Its been advised to use Mocha for WDIO(even it supports Jasmine, Cucumber), is there any specific advantage of using Mocha over Jasmine?
Also, for WDIO config, default FW option is mocha and it has below default option
mochaOpts: {
        ui: 'bdd',
        }, 
whereas I could able to develop execute my tests in non BDD/Gherkins format as below, so below is TDD format or BDD format? 

Test example:

describe('test',async()=>
{
      
    it('BrowserTest1',async()=>
{
   
    await   browser.url("https://google.com")
 
  })

})

**Comment by christian-bromann** [comment]
> is there any specific advantage of using Mocha over Jasmine?

IMO Mocha has better retry mechanism but Jasmine has some other advantages too.

> so below is TDD format or BDD format?

This would be more `tdd`, see more https://mochajs.org/#interfaces
